pages/en/lb3/Polymorphic-relations.md

```diff
@@ -34,9 +34,9 @@ This documentation is still a work in progress.
 " %}
 
 LoopBack supports _polymorphic relations_ in which a model can belong to more than one other model, on a single association.
-For example, you might have a Picture model that belongs to either an Author model or a Reader model. 
+For example, you might have a Picture model that belongs to either an Author model or an Reader model. 
```

Comment:
good catch. At first I use `article` and `employee` as example models, left some `an` before model name, will clean up those places.